diff --git a/dist/esm/src/utils/helpers.js b/dist/esm/src/utils/helpers.js
index 408c7b0e2a0e90bb96b25fb82c86bec94d0d337e..21127b6ec169d712e4a39f2766f6bb640a71a789 100644
--- a/dist/esm/src/utils/helpers.js
+++ b/dist/esm/src/utils/helpers.js
@@ -39,7 +39,7 @@ export function parseWalletCapabilities(str) {
 }
 export async function getSafeConnector(connectors) {
     if (CoreHelperUtil.isSafeApp()) {
-        const { safe } = await import('@wagmi/connectors');
+        const { safe } = await import('@wagmi/connectors/safe');
         if (safe && !connectors.some(c => c.type === 'safe')) {
             const safeConnector = safe();
             return safeConnector;
@@ -47,21 +47,12 @@ export async function getSafeConnector(connectors) {
     }
     return null;
 }
-export async function getBaseAccountConnector(connectors) {
-    try {
-        const { baseAccount } = await import('@wagmi/connectors');
-        if (baseAccount && !connectors.some(c => c.id === 'baseAccount')) {
-            return baseAccount();
-        }
-    }
-    catch (error) {
-        console.error('Failed to import Base Account SDK:', error);
-    }
+export async function getBaseAccountConnector(_connectors) {
     return null;
 }
 export async function getCoinbaseConnector(connectors, preference) {
     try {
-        const { coinbaseWallet } = await import('@wagmi/connectors');
+        const { coinbaseWallet } = await import('@wagmi/connectors/coinbaseWallet');
         if (coinbaseWallet && !connectors.some(c => c.id === 'coinbaseWallet')) {
             return coinbaseWallet({ preference });
         }
